src/ride/ride.c

```diff
@@ -124,6 +124,438 @@ uint8 gTypeToRideEntryIndexMap[TYPE_TO_RIDE_ENTRY_SLOTS];
 
 #pragma endregion
 
+const rct_ride_properties gRideProperties[91] = {
+		{RIDE_ENTRY_FLAG_0 | RIDE_ENTRY_FLAG_NO_INVERSIONS | RIDE_ENTRY_FLAG_NO_BANKED_TRACK |
```

Comment:
Hmm, you may be right.

In that case, I was misled by this comment:
```
// Constants used by the ride_type->flags property at 0x008
```
https://github.com/janisozaur/OpenRCT2/blob/integrate2/src/ride/ride.h#L413